src/react-components/package.json

```diff
@@ -6,14 +6,16 @@
     "type": "git",
     "url": "https://github.com/NYPL/nypl-design-system/tree/development/src/react-components"
   },
-  "main": "lib/index.js",
+  "main": "dist/design-system-react-components.min.js",
   "types": "lib/index.d.ts",
   "author": "NYPL",
   "license": "Apache-2.0",
   "scripts": {
     "prepare": "npm run build",
-    "prepublishOnly": "npm run test",
+    "prepublishOnly": "npm run lint && npm run test && npm run dist",
```

Comment:
nit: can take out `npm run lint` because it is already run in `npm run test` -- dw if it's already ready to merge tho